pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.targomo</groupId>
     <artifactId>java-client</artifactId>
-    <version>0.1.9-SNAPSHOT</version>
+    <version>0.1.10-SNAPSHOT</version>
```

Comment:
we're only at 0.1.9-SNAPSHOT - the last release was 0.1.8

Reply:
The statistic service is already using `0.1.10-SNAPSHOT`, so I think going back a version is worse then leaping one forward.